packages/popper/src/use-popper/build-modifiers.ts

```diff
@@ -41,7 +43,8 @@ export default function buildModifier(props: ModifierProps, externalModifiers: S
     {
       name: 'computeStyles',
       options: {
-        adaptive: true,
+        gpuAcceleration,
+        adaptive: gpuAcceleration,
```

Comment:
These two were not conflicted?

Reply:
When set `gpuAcceleration` to false, adaptive should also be false, otherwise will get a warning.

Reply:
Got it